spec/models/review_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe Review, type: :model do
+  let(:review) { Review.new(name: "Arnold Schwarzenegger",
+                            content: "Ahftah trahvelig with Endangerous Ouhtfittahs, I hahve one thing to sahy...I'll be bahck.",
+                            occupation: "Movie Star, Action Hero, Governator, Terminator, Mr. Universe", image_url: "") }
+
+  it 'is valid' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.